sdk/storage/Azure.ResourceManager.Storage/CHANGELOG.md

```diff
@@ -10,6 +10,10 @@
 
 ### Other Changes
 
+## 1.4.3 (2025-06-24)
+
+- Updated per swagger change for StorageAction GA
```

Comment:
This line should be place under '### Features Added' section rather than under the version heading directly.

Reply:
fixed